liquibase-core/src/main/java/liquibase/license/Location.java

```diff
@@ -14,82 +14,37 @@
 public class Location {
```

Comment:
Are we able to get rid of this entire Location class?

Reply:
I think we'll still need the location class, because the license service requires knowing both the value of the license, and metadata about where the license came from. I did simplify the license class a bit.